tests/conftest.py

```diff
@@ -0,0 +1,139 @@
+import aiomcache
+import aioredis
+import asyncio
+import gc
+import pytest
+import time
+import uuid
+from docker import Client as DockerClient
+
+
+@pytest.yield_fixture(scope='session')
```

Comment:
I pretty sure `yield_fixture` could be replaced by just `@pytest.fixture` in modern pytest.
We still have `yield_fixture` in very many *aio-libs* projects.
Need for a hero for replace them all or ending of deprecation period for `@pytest.fixture` :)

Reply:
I can go through a few of them (probably next weekend) :)